src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,18 @@
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <>
+    <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''} `}>
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      {todo.user && <UserInfo user={todo.user} />}
+    </article>
+  </>
```

Comment:
You don't need an additional fragment here
```suggestion
    <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''} `}>
      <h2 className="TodoInfo__title">{todo.title}</h2>

      {todo.user && <UserInfo user={todo.user} />}
    </article>
```